iOS10-HW14-Bessonov Ilia/Sources/ViewController.swift

```diff
@@ -0,0 +1,294 @@
+                                                      heightDimension: .fractionalHeight(1))
+                let layoutItem = NSCollectionLayoutItem(layoutSize: itemSize)
+                layoutItem.contentInsets = NSDirectionalEdgeInsets(top: 5,
+                                                                   leading: 5,
+                                                                   bottom: 5,
+                                                                   trailing: 0)
+
+                let groupSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1),
+                                                       heightDimension: .absolute(44))
+                let layoutGroup = NSCollectionLayoutGroup.horizontal(layoutSize: groupSize,
+                                                                     subitems: [layoutItem])
+
+                let layoutSection = NSCollectionLayoutSection(group: layoutGroup)
+                layoutSection.contentInsets = NSDirectionalEdgeInsets(top: 50,
+                                                                      leading: 10,
+                                                                      bottom: 50,
+                                                                      trailing: 10)
+
+                return layoutSection
+            }
+        }
+    }
+}
+
+// MARK: - CollectionViewDelegate
+
+extension ViewController: UICollectionViewDataSource, UICollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+            return PhotosModel.modelsArray[section].count
+    }
+
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        switch indexPath.section {
+        case 0:
+            let item = collectionView.dequeueReusableCell(withReuseIdentifier: MyAlbumsCell.identifier, for: indexPath) as! MyAlbumsCell
+            item.configuration(model: PhotosModel.modelsArray[indexPath.section][indexPath.item])
+
+            return item
+        case 1:
+            let item = collectionView.dequeueReusableCell(withReuseIdentifier: SharedAlbumsCell.identifier, for: indexPath) as! SharedAlbumsCell
+            item.configuration(model: PhotosModel.modelsArray[indexPath.section][indexPath.item])
+
+            return item
+        case 2:
+            let item = collectionView.dequeueReusableCell(withReuseIdentifier: TableCellInCollectionCell.identifier, for: indexPath) as! TableCellInCollectionCell
+            item.configuration(model: PhotosModel.modelsArray[indexPath.section][indexPath.item])
+
+            return item
+        case 3:
+            let item = collectionView.dequeueReusableCell(withReuseIdentifier: TableCellInCollectionCell.identifier, for: indexPath) as! TableCellInCollectionCell
+            item.configuration(model: PhotosModel.modelsArray[indexPath.section][indexPath.item])
```

Comment:
И Кастить лучше через гард так безопасней 
```suggestion
             guard let item = collectionView.dequeueReusableCell(
                withReuseIdentifier: TableCellInCollectionCell.identifier,
                for: indexPath) as? TableCellInCollectionCell else {
                return UICollectionViewCell()
            }
```